packages/neutrino/src/webpack.js

```diff
@@ -25,9 +25,14 @@ const compile = pipe(
 // devServer :: Object config -> Future Error Object
 const devServer = pipe(
   compiler,
-  Future.map(compiler => assoc('compiler', compiler, new DevServer(compiler, compiler.options.devServer)))
+  Future.map((compiler) => {
```

Comment:
This is essentially the same thing that was happening before, except that `assoc` was:

> Makes a shallow clone of an object, setting or overriding the specified property with the given value. Note that this copies and flattens prototype properties onto the new object as well. All non-primitive properties are copied by reference.

This was probably messing with the DevServer instance. Switch to an `Object.assign` instead:

```js
Future.map(compiler => Object.assign(new DevServer(compiler, compiler.options.devServer), { compiler }))
```